plugins/bgp/model/bgp.proto

```diff
@@ -7,13 +7,13 @@ package model;
 
 //BGP configuration
 message BgpConf {
-    Global global = 1;
```

Comment:
It is not needed to define "parent" model wrapping all the models below. Configuration will be passed to the agent (using KVDB or gRPC) directly through the GlobalConf  or PeerConf-s (i.e. the models for which the descriptors are defined).
We have a similar situation for NAT: https://github.com/ligato/vpp-agent/blob/master/api/models/vpp/nat/nat.proto
There is a global configuration `Nat44Global` and a set of instances modeled with `DNat44`, each with its own label (in your case PeerConf.Name), but there is no model wrapping both of them.

Reply:
We went over this with Jan and Nikos, and they want to keep the BgpConf since we might need it at a later point. But we will keep this in mind, thank you!